### ISSUE TITLE
[Warning][Django 1.10] Reporting warnings in jet.dashboard

### ISSUE THREAD

**Issue by SalahAdDin** [geex-arts/django-jet#42]
I want report the next warnings:

```bash
/run/media/salahaddin/Data/Proyectos/Trabajo/ccct/lib/python3.5/site-packages/jet/urls.py:31: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  name='jsi18n'

/run/media/salahaddin/Data/Proyectos/Trabajo/ccct/lib/python3.5/site-packages/jet/dashboard/dashboard.py:6: RemovedInDjango110Warning: django.core.context_processors is deprecated in favor of django.template.context_processors.
  from django.core.context_processors import csrf

/run/media/salahaddin/Data/Proyectos/Trabajo/ccct/lib/python3.5/site-packages/jet/dashboard/urls.py:49: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  name='jsi18n'
```

**Comment by coolharsh55** [comment]
Pull-request #30 addresses this (not merged yet, build failing).

**Comment by f1nality** [comment]
Thanks! merged in 0.1.5
